src/js/Attachment/__test__/setFiles.test.ts

```diff
@@ -0,0 +1,50 @@
+/* eslint-disable @typescript-eslint/no-empty-function */
+import Attachment from '../index';
+
+describe('Unit test Attachment setFiles', () => {
+  beforeEach(() => {
+    jest.spyOn(console, 'error');
+    // @ts-ignore
+    console.error.mockImplementation(() => { });
+  });
+  afterEach(() => {
+    // @ts-ignore
+    console.error.mockRestore();
+  });
+
+  test('setFiles is called successfully', () => {
+    const myAttachment = new Attachment();
+    const FILE_NAME = 'test_1';
+    const FILE_SIZE = 1024;
+    const FORMATED_FILE_SIZE = '1 KB';
+    myAttachment.setFiles([{name: FILE_NAME, size: FILE_SIZE}]);
+
+    // Verify files prop
+    const files = myAttachment.getFiles();
+    expect(files).toBeInstanceOf(Array);
+    expect(files!.length).toEqual(1);
+    expect(files![0]).toHaveProperty('name');
+    expect(files![0]).toHaveProperty('size');
+    expect(files![0].name).toEqual(FILE_NAME);
+    expect(files![0].size).toEqual(FILE_SIZE);
```

Comment:
Line no need:
expect(files![0]).toHaveProperty('name');
expect(files![0]).toHaveProperty('size');

Reply:
Line 26 and 27 is used verify `name` and `size` property existed before checking the value.